### PR TITLE
Feature/update static contents

### PR DESCRIPTION
**Pull request by ugudlado**
- Changed title to upper case as Azhagu suggested
- Add Idea page
- Added advisory page

### CODE REVIEW

File: src/app/app.service.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ export class AppService {
   public allUsers: any[] = [];
   public isLoading: boolean = false;
   public materials = MATERIALS;
+  public title = 'INDIA WASTE EXCHANGE';
```
Comment (subbu3012) [inline src/app/app.service.ts]: why is this added in `app.service.ts`? it should have been added in `app.component.ts` and accessed in `app.component.html` as `{{ this.title }}`.

File: src/app/shared/components/info/info.component.html [hunk 2]
```diff
@@ -137,3 +143,106 @@ <h3 class="text--center text--blue">Special Thanks</h3>
     </div>
   </div>
 </ng-template>
+
+<ng-template #ideaTemplate>
+  <div class="our-team-container">
```
Comment (subbu3012) [inline src/app/shared/components/info/info.component.html]: Please change this class name and subsequent class names to `idea-container` or something. also, add child divs only if needed.

File: src/app/shared/components/info/info.component.html [hunk 3]
```diff
@@ -137,3 +143,106 @@ <h3 class="text--center text--blue">Special Thanks</h3>
...
+      <p class="justified-text">
+        This Waste Exchange platform will be scaled up across all 100 Smart Cities in India. This will create a huge set of micro networks
+        in waste management supply chain across India to communicate among each other. This in turn will help country to achieve net zero
+        waste.
+      </p>
+    </div>
+  </div>
+</ng-template>
+<ng-template #advisoryTeamTemplate>
+  <div class="special-thanks-container">
```
Comment (subbu3012) [inline src/app/shared/components/info/info.component.html]: Please change thi class name and subsequent class names to `advisory-team-container` or something.

File: src/app/shared/components/info/info.component.html (hunk 2 above)
Comment (vraravam) [reply src/app/shared/components/info/info.component.html]: @subbu3012 - you mean to use hyphenated instead of camelCase?

Comment (subbu3012) [reply src/app/shared/components/info/info.component.html]: @vraravam no. the container's class name should be `idea-container` instead of `our-team-container` since the template is `ideaTemplate`.

File: src/app/app.service.ts (hunk 1 above)
Comment (ugudlado) [reply src/app/app.service.ts]: I think component as controller and should not do or have anything other than mediator between view and models via services.

I am not sure how you are following here. So, I will make change here

File: src/app/shared/components/info/info.component.html (hunk 2 above)
Comment (ugudlado) [reply src/app/shared/components/info/info.component.html]: Done. I was following same layout as other containers in same Info component.

File: src/app/shared/components/info/info.component.html (hunk 3 above)
Comment (ugudlado) [reply src/app/shared/components/info/info.component.html]: Done.

**Comment by ugudlado** [comment]
> Please fix the review comments.

Please check and let me know if you have more comments.
